1_5_3 and 1_5_4/includes/templates/YOUR_TEMPLATE/templates/tpl_modules_main_product_image.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
I do understand that.  Seeing though that additional information is populated below it to identify that SBA has been incorporated it would be here that one could indicate that would work with each version, etc...  Afterall, the plugin has been developed to work with these versions.  Otherwise, might as well return to the three/four/five/etc different repos such that each has just the files associated with that version of ZC...